samples/drivers/led_max7219/prj.conf

```diff
@@ -0,0 +1,10 @@
+# nothing here
+CONFIG_NO_OPTIMIZATIONS=y
```

Comment:
Why do we need to enforce no optimizations?

Reply:
Actually we don't. I had this project config laying around and just poked around it until SPI worked as expected. I forgot to come back and clean it up. Fixed.